src/app/admin/admin.component.ts

```diff
@@ -1244,7 +1244,7 @@ export class AdminComponent implements OnInit {
               this.globalVars._alertSuccess(
                 sprintf(
                   'Successfully updated the reserve exchange to $%d/DeSo',
-                  res.USDCentsPerDeSo / 100
+                  res.USDCentsPerDeSoCoinbase / 100
```

Comment:
Revert this one please - it the response from this endpoint returns `USDCentsPerDeSo`